english/watagatapistsusberry/watagatapitsusberry_Fruit.rtf

```diff
@@ -0,0 +1 @@
+Berry with cranberry, you're feeling so hairy, maybe you're spaghetti? Those are the side effects of eating the watagatapitsusberry fruit! 
```

Comment:
It looks like this page is not reachable from the rest of the story. Make you sure you add a link to an existing page that leads to this page, or readers will not be able to find your addition.